doc/api/crypto.md

```diff
@@ -1671,6 +1671,110 @@ Use [`crypto.getHashes()`][] to obtain an array of names of the available
 signing algorithms. Optional `options` argument controls the
 `stream.Writable` behavior.
 
+### crypto.generateKeyPair(type, options, callback)
+<!-- YAML
+added: REPLACEME
+-->
+* `type`: {string} Must be `'rsa'`, `'dsa'` or `'ec'`.
```

Comment:
Nit: there are some inconsistencies in upper/lower case and periods in all list items. Maybe some native speakers can chime in to normalize?